src/vl-upload.js

```diff
@@ -247,6 +247,10 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
       this._dressFormValidation();
       this._dropzone.on('addedfile', () => setTimeout(() => this.dispatchEvent(new Event('change'))));
       this._dropzone.on('removedfile', () => setTimeout(() => this.dispatchEvent(new Event('change'))));
+      this._dropzone.on('success', (file, response) => {
+      	file.responseData = response;
```

Comment:
is dat eigenlijk de originele http response dat je hier krijgt, of de body van de response?

Reply:
Als ik de testen lees is dat precies de body, dus.

Reply:
miss hernoemen naar responseBody dan is het meteen duidelijk

Reply:
redelijk equivalent zou kik zeggen maar wil het wel aanpassen